src/components/deploymentConfig/DeploymentTemplateView/DeploymentTemplateView.component.tsx

```diff
@@ -296,16 +277,111 @@ export const renderEditorHeading = (
                         className={`cursor ${overridden ? 'cr-5' : 'cb-5'}`}
                         onClick={handleOverride}
                     >
-                        {overridden ? 'Delete override' : 'Allow override'}
+                        {isValues?(overridden ? 'Delete override' : 'Allow override'):''}
                     </span>
                 )}
             </div>
         </div>
     )
-}
 
-export const SuccessToastBody = ({ chartConfig }) => {
+export const CompareWithApprovalPendingAndDraft = ({
+    isEnvOverride,
+    overridden,
+    readOnly,
+    environmentName,
+    selectedChart,
+    handleOverride,
+    latestDraft,
+    isPublishedOverriden,
+    isDeleteDraftState,
+    setShowDraftData,
+    isValues
+}: CompareWithApprovalPendingAndDraftProps) => {
+
+    const APPROVAL_PENDING_OPTION = { id: 0, label: 'Approval Pending' };
+    const DRAFT_OPTION = { id: 1, label: `${isValues?'Values':'Manifest'} from draft` }
+
+    const [selectedOption, setSelectedOption] = useState(APPROVAL_PENDING_OPTION)
+
+    const options = [
```

Comment:
Please move this to constant and rename it to more specific

Reply:
done